src/js/helpers/isMobile.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+/**
+ * Returns whether browser is a mobile or not. Tests for touch support and
+ * screen width.
+ *
+ * @returns {boolean} True if mobile.
+ */
+module.exports = function () {
+  // If doesn't support touch
+  if (!('ontouchstart' in window) && !navigator.msMaxTouchPoints) {
```

Comment:
Use https://www.npmjs.com/package/touch-screen. Where did you take this from? I'll update it there, too.